I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 33, in script
    define config.main_menu = [
  File "game/script.rpy", line 38, in <module>
    ("Quit", _quit, "True")
NameError: name '_quit' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 33, in script
    define config.main_menu = [
  File "E:\renpy\renpy-8.3.0-sdk\renpy\ast.py", line 2234, in execute
    self.set()
  File "E:\renpy\renpy-8.3.0-sdk\renpy\ast.py", line 2248, in set
    value = renpy.python.py_eval_bytecode(self.code.bytecode)
  File "E:\renpy\renpy-8.3.0-sdk\renpy\python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 38, in <module>
    ("Quit", _quit, "True")
NameError: name '_quit' is not defined

Windows-10-10.0.22000 AMD64
Ren'Py 8.3.0.24082114
111 1.0
Fri Oct  4 19:14:51 2024
